packages/element-plus/package.json

```diff
@@ -20,9 +20,9 @@
   "types": "es/index.d.ts",
   "exports": {
     ".": {
+      "types": "./es/index.d.ts",
       "require": "./lib/index.js",
-      "import": "./es/index.mjs",
-      "types": "./es/index.d.ts"
+      "import": "./es/index.mjs"
```

Comment:
It may be better to put `import` in front of `require`, and support `esm` first

Reply:
It looks good 👍